hbase-server/src/main/java/org/apache/hadoop/hbase/replication/regionserver/ReplicationSourceWALReader.java

```diff
@@ -151,6 +151,7 @@ public void run() {
             } else {
               addBatchToShippingQueue(batch);
             }
+            sleepMultiplier = 1;
```

Comment:
Good to keep this after `while (isReaderRunning()) {` perhaps?
If we encounter InterruptedException while sleeping if peer is not enabled, we might skip this reset? But on the other hand, after getting interrupted, we are not dealing with this multiplier also. But if some other Exception is caught before reaching here, this reset won't apply.

Reply:
This one is just a nit, and not that much of a concern.